packages/blocks/package.json

```diff
@@ -23,7 +23,7 @@
     "@fal-ai/serverless-client": "^0.10.0",
     "@floating-ui/dom": "^1.6.5",
     "@sgtpooki/file-type": "1.0.1",
-    "@toeverything/theme": "^0.7.29",
+    "@toeverything/theme": "^0.7.30",
```

Comment:
The latest commit is not included. Did you forget to release a new one?

Reply:
Now I've re-synced with `gt sync`, seems no conflict.

Reply:
I mean the latest commit in @toeverything/theme https://github.com/toeverything/design/commit/04adbc241d39bc2e8e7b521695a5153527c84c38

Reply:
This commit currently has no impact on functionality. The main purpose of bumping the theme is to unset the button's inner shadow.